code-of-conduct.md

```diff
@@ -0,0 +1,3 @@
+# Kubernetes Community Code of Conduct
+
+Please refer to our [Kubernetes Community Code of Conduct](https://git.k8s.io/community/code-of-conduct.md)
```

Comment:
minor nit - add trailing period? :)

Reply:
This is a template so it is the exact same in all k/* repos (ref: https://github.com/kubernetes/community/issues/1527)